README.md

```diff
@@ -39,4 +39,73 @@ We always have an opportunity to make right our mistake and refund the customer
 We are currently working on the implementation and a set of standards to minimise integration efforts with us. If you want to partner with us such that your customers can hire PISA to respond on their behalf - please contact us at paddy@pisa.watch and check out the following standards (we will update this list as more are posted):
 
 * Data Registry (log events) - https://github.com/ethereum/EIPs/pull/2095 
-* Example of contract logging events (super simple) - https://github.com/PISAresearch/pisa/blob/master/sol/contracts/ChallengeClosureContract.sol 
\ No newline at end of file
+* Example of contract logging events (super simple) - https://github.com/PISAresearch/pisa/blob/master/sol/contracts/ChallengeClosureContract.sol 
+
+# Installation
+Clone the repository and install the requirements:
+```
+git clone https://github.com/PISAresearch/pisa.git
+cd pisa
+npm install
+```
+
+## Run tests
+
+```
+npm run test               # run unit and end-to-end tests
+npm run test-unit          # only run unit tests
+npm run test-endtoend      # only run end-to-end tests
+npm run test-integration   # run integration-tests
+npm run test-contract      # run smart contract's unit tests
+```
+
+## Build production instance
+
+```
+npm run build              # build PISA
+npm run build-client       # build the client library
+```
+
+# Hiring Pisa
+
+**Note:** Pisa is alpha stage software and is not yet available on mainnet. Do not use with real funds.
```

Comment:
Maybe also write that all specifications and APIs are subject to change at any time

Reply:
Done.